PowerShellTools.HostService/ServiceManagement/IntelliSense/PowershellIntelliSenseService.cs

```diff
@@ -104,6 +104,17 @@ public void RequestCompletionResults(string script, int caretPosition, long trig
         }
 
         /// <summary>
+        /// Suspecting this is a powershell bug, the first time you call CommandCompletion.CompleteInput, it takes much longer than usual.
+        /// We are using this dummy call during intializing to warm it up.
+        /// </summary>
+        /// <param name="scriptUpToCaret"></param>
+        /// <param name="carePosition"></param>
+        public void GetDummyCompletionList(string scriptUpToCaret, int carePosition)

```

Comment:
Since we always plan to call the dummy completion for dummy purposes, do we want to change the signature to GetDummyCompletionList() with the Write- and 6 hardcoded?